fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -223,3 +220,530 @@ fn expand_input_param(
         _ => expand_type(kind),
     }
 }
+
+// Regarding string->TokenStream->string, refer to `custom_types_gen` tests for more details.
+#[cfg(test)]
+mod tests {
```

Comment:
I'm a bit confused, aren't these tests already being introduced in https://github.com/FuelLabs/fuels-rs/pull/73?

Reply:
I just built #73 on top of #67 haha sorry I should have waited

Reply:
Hah, it happens. You can close this one and we continue the review process on https://github.com/FuelLabs/fuels-rs/pull/73?

Reply:
Or maybe, since you wanna merge this one first, you can address my comments on https://github.com/FuelLabs/fuels-rs/pull/73 here, then ping me when you want me to take another pass?

Reply:
I'm going to fix everything at once in #73.

Reply:
Sounds good!